lib/proc-macros/src/lib.rs

```diff
@@ -6,6 +6,8 @@ use quote::quote;
 use syn::parse_macro_input;
 use syn::DeriveInput;
 
+/// Sequentially hash struct members to derive `ContentHash` trait
+///
```

Comment:
nit: it also supports enum types. Maybe just copy the std doc phrase?
https://doc.rust-lang.org/std/fmt/derive.Debug.html